Holovibes/includes/gui/windows/Filter2DWindow.hh

```diff
@@ -12,7 +12,7 @@ class MainWindow;
 
 /*! \class Filter2DWindow
  *
- * \brief #TODO Add a description for this class
+ * \brief class that represents a Filter2D window in the GUI.
```

Comment:
**C**lass
Similarly to the overlays, I think the file description could do better as the class description but then again it's a personal preference. U choose

Reply:
I won't do like the overlays again, but this goes for the other windows as well